storage_drivers/eseries_iscsi.go

```diff
@@ -57,14 +57,16 @@ func (d *ESeriesStorageDriver) Initialize(configJSON string) error {
 
 	d.config = *config
 	d.storage = eseries.NewDriver(eseries.DriverConfig{
-		WebProxy_Hostname: config.WebProxy_Hostname,
-		Username:          config.Username,
-		Password:          config.Password,
-		Controller_A:      config.Controller_A,
-		Controller_B:      config.Controller_B,
-		Password_Array:    config.Password_Array,
-		Array_Registered:  config.Array_Registered,
-		HostData_IP:       config.HostData_IP,
+		WebProxyHostname: config.WebProxyHostname,
+		WebProxyPort:     config.WebProxyPort,
+		WebProxyInsecure: config.WebProxyInsecure,
+		Username:         config.Username,
+		Password:         config.Password,
+		ControllerA:      config.ControllerA,
+		ControllerB:      config.ControllerB,
+		PasswordArray:    config.PasswordArray,
+		ArrayRegistered:  config.ArrayRegistered,
+		HostDataIP:       config.HostData_IP,
```

Comment:
Possibly change config.HostData_IP to config.HostDataIP to make it consistent with some of the other variable name changes you have done.